### ISSUE TITLE
femur and tibia compact bone volume

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3807]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. decreased femoral compact bone volume
2. decreased tibial compact bone volume
3. increased femoral compact bone volume
4. increased tibial compact bone volume

>2. Synonyms (e.g., absent spleen)

1. reduced femoral compact bone volume; decreased femur compact bone volume; reduced femur compact bone volume; decreased femur cortical bone volume
2. reduced tibial compact bone volume; decreased tibia compact bone volume; reduced tibia compact bone volume; decreased tibia cortical bone volume
3. increased femur compact bone volume; increased femur cortical bone volume
4. increased tibial compact bone volume; increased tibia cortical bone volume

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. less than normal amount of space occupied by compact bone tissue in the femur
2. less than normal amount of space occupied by compact bone tissue in the tibia
3. greater than normal amount of space occupied by compact bone tissue in the femur
4. greater than normal amount of space occupied by compact bone tissue in the tibia

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:35239186

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. decreased compact bone volume (MP:0010965)
2. decreased compact bone volume (MP:0010965)
3. increased compact bone volume (MP:0010964)
4. increased compact bone volume (MP:0010964)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by sbello** [comment]
new terms
decreased femur compact bone volume MP:0014307
decreased tibia compact bone volume MP:0014308
increased femur compact bone volume MP:0014309
increased tibia compact bone volume MP:0014310
